### ISSUE TITLE
Compatible with App Runtime for Chrome?

### ISSUE THREAD

**Issue by tristan-k** [Eloston/ungoogled-chromium#12]
Is it possible run android apps with the App Runtime for Chrome? There is [ARC-Welder](https://chrome.google.com/webstore/detail/arc-welder/emfinbmielocnlhgmfkkmkngdoccbadn) as well as the [ARChon Runtime for Chrome](https://archon-runtime.github.io/). 

I tried to install ARC-Welder but it fails with the error-message `CRX_HEADER_INVALID`. 

I installed ARChon and loaded the [sample app](https://github.com/vladikoff/chromeos-apk/releases/download/v3.0.0/com.uberspot.a2048.android-ARChon-runtime.zip) but it freezes while loading. 

In the debug messages I can see that ARChon is complaining about:

```
'minimum_chrome_version' is only allowed for extensions, hosted apps, legacy packaged apps, and packaged apps, but this is a shared module.
Unrecognized manifest key 'arc_build_tag'
```

And the sample app is complaining about:
 
```
Unrecognized manifest key 'arc_metadata'.
```

**Comment by Eloston** [comment]
I don't have a way of downloading ARC-Welder; so I can't help you with that.

I noticed that ARChon uses NaCl. I disabled this in Debian builds (with the GYP flag `disable_nacl=1`). Did you build ungoogled-chromium without NaCl support?

IIRC, building with NaCl requires additional downloading during the GYP configuration or ninja steps, so you'll need to undo some changes made by `domain_patcher.sh`.
